app/controllers/schedules_controller.rb

```diff
@@ -4,7 +4,7 @@ class SchedulesController < ApplicationController
 
   def new
     @schedule = @guider.schedules.build(
-      start_at: 6.weeks.from_now
+      start_at: 6.weeks.from_now + 1.day
```

Comment:
I think we should move this logic into `User` or perhaps even within the scope of `User#schedules` (you can pass a block of methods that are executed in that scope.) 

It seems like a domain thing that should be enforced and is somewhat lost out here in this controller.